### ISSUE TITLE
is_impersonated_session is not working for multiple level log in as

### ISSUE THREAD

**Issue by juancresc** [skorokithakis/django-loginas#105]
I have different user types:
admin
manager
regular user

I can go through different levels using log in as, but once I click loginas-logout, in the first level, then I cannot go up back to admin.
I wonder if loginas-logout should go to the original user

**Comment by skorokithakis** [comment]
Hm, do you not go back to manager?

**Comment by juancresc** [comment]
Basically

I log in as admin
Then I impersonate a manager
Then a regular user

Then, loginas-logout takes me to manager, but then the is_impersonated_session is false and I cannot go back to admin

**Comment by skorokithakis** [comment]
Yes, that's as designed, you currently can only go back one level. I wouldn't be against a PR that lets you nest, though.
